Dockerfile

```diff
@@ -28,6 +28,3 @@ RUN pip install -e .
 RUN adduser -D worker
 RUN chown worker:worker -R /home/worker
 USER worker
-
-ENTRYPOINT ["chanjo"]
```

Comment:
I've tried for many hours to use the image with the preset chanjo command. I couldn't make it work because if you set the entrypoint, then you can't even access the bash commands inside the container. I needed the bash commands because the database setup is done in 2 steps:

- `chanjo init --auto demodata` --> creates a folder where the exons and config files are saved
- `chanjo --config demodata/chanjo.yaml link demodata/hgnc.grch37p13.exons.bed` -> extracts the exons from the step before and saves them to database, reads from config file in demodata
 
If you run the 2 commands into different containers the demodata folder exists only in the first one and the second command can't find the required files.

It can probably be fixed in some other way, but I've spent a lot of time on it and this is the only way I could make it work --> by running the 2 commands in the same container (see this step in the Makefile)